pkg/internal/cnsoperator/cnsfilevolumeclient/cnsfilevolumeclient.go

```diff
@@ -35,6 +35,9 @@ import (
 // FileVolumeClient exposes an interface to support
 // configuration of CNS file volume ACL's.
 type FileVolumeClient interface {
+	// GetClientVmsFromIPList returns the list of client vms associated with a given External IP address
+	// and a given CnsFileVolumeClient instance name
+	GetClientVmsFromIPList(ctx context.Context, fileVolumeName string, clientVMIP string) ([]string, error)
```

Comment:
`GetClientVmsFromIPList` => `GetClientVMsFromIPList` for consistency with other functions of this interface